proto/desmos/profiles/v1beta1/genesis.proto

```diff
@@ -30,4 +30,12 @@ message GenesisState {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"params\""
   ];
+
+  string port_id = 5 [
```

Comment:
Maybe we can rename this to be `ibc_port_id` to make it more clear this is IBC-related?